src/main/cpp/src/get_json_object.cu

```diff
@@ -199,13 +542,15 @@ __device__ parse_result parse_json_path(json_parser<max_json_nesting_depth>& j_p
  */
 template <int max_json_nesting_depth = curr_max_json_nesting_depth>
 __device__ thrust::pair<parse_result, json_generator<max_json_nesting_depth>>
-get_json_object_single(char const* input,
-                       cudf::size_type input_len,
-                       path_instruction const* path_commands_ptr,
-                       int path_commands_size,
-                       char* out_buf,
-                       size_t out_buf_size,
-                       json_parser_options options)
+get_json_object_single(
+  char const* input,
+  cudf::size_type input_len,
+  path_instruction const* path_commands_ptr,
+  int path_commands_size,
+  char* out_buf,
+  size_t out_buf_size,
+  json_parser_options options)  // TODO make this a reference? use a global singleton options?
```

Comment:
Or just don't have a json_parser_options at all. We don't have a way to change them, so why not just hard code it?

Reply:
OK, I'll hard code it.